OpenRA.Game/Sound/Sound.cs

```diff
@@ -247,6 +247,21 @@ public void PlayMusic()
 			soundEngine.PauseSound(music, false);
 		}
 
+		public void PlayMusicLooped(MusicInfo m)
```

Comment:
Since this method is effectively the latter half of `PlayMusicThen` with a parameter flipped - is it worth extracting a private method to share this code?

Reply:
I refactored this.